infra/modules/main/backend.tf

```diff
@@ -7,7 +7,20 @@ resource "aws_s3_bucket" "backend_code" {
     target_bucket = var.s3_logs_bucket
     target_prefix = "${var.name_prefix}-backend-code/"
   }
+}
 
+locals {
```

Comment:
I guess this is good, removing duplication of environment variable declarations 👌

Reply:
Yeah, that was the idea! So we're less likely to update API env but forget making the corresponding change to worker env, or vice versa.